sql/_units.sql

```diff
@@ -98,7 +98,7 @@ SELECT
 			OR (classa_prop IS NOT NULL AND classa_prop <> '0')
 			THEN 'Residential' 
 	END) as resid_flag
-INTO _UNITS_devdb
+INTO _UNITS_devdb_resid_flag
```

Comment:
if the intention here is to replace `_UNITS_devdb` with `_UNITS_devdb_resid_flag` there are a few more places in this sql file still referenced to `_UNITS_devdb`.

Reply:
upon a further look only line 111 needs adjustment the other reference seems okay to me.